.github/workflows/ci.yml

```diff
@@ -168,15 +168,8 @@ jobs :
           java-version : '11'
           check-latest : true
 
-      # TODO: Revert release changes when we drop support for 1.7
-      - name : Publish Snapshots 1.7
-        run : ./gradlew clean publish --no-build-cache --no-daemon --stacktrace --no-parallel -Psquare.kotlinVersion=1.7.20 && cd gradle-plugin && ./gradlew clean publish --no-build-cache --no-daemon --stacktrace -Psquare.kotlinVersion=1.7.20 && cd ..
-        env :
-          ORG_GRADLE_PROJECT_mavenCentralUsername : ${{ secrets.SONATYPE_NEXUS_USERNAME }}
-          ORG_GRADLE_PROJECT_mavenCentralPassword : ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
-
-      - name : Publish Snapshots 1.8
-        run : ./gradlew clean publish --no-build-cache --no-daemon --stacktrace --no-parallel -Psquare.kotlinVersion=1.8.0 -PVERSION_NAME=2.4.4-1-8-0-SNAPSHOT && cd gradle-plugin && ./gradlew clean publish --no-build-cache --no-daemon --stacktrace -Psquare.kotlinVersion=1.8.0 -PVERSION_NAME=2.4.4-1-8-0-SNAPSHOT && cd ..
+      - name : Publish Snapshots
+        run : ./gradlew clean publish --no-build-cache --no-daemon --stacktrace --no-parallel -PVERSION_NAME=2.4.4 && cd gradle-plugin && ./gradlew clean publish --no-build-cache --no-daemon --stacktrace -PVERSION_NAME=2.4.4 && cd ..
```

Comment:
I think VERSION_NAME should be left off here right? Or updated to inclue -SNAPSHOT

Reply:
Good catch, I removed the version name.